DatadogRUM/Tests/Instrumentation/Views/RUMViewsHandlerTests.swift

```diff
@@ -126,11 +126,11 @@ class RUMViewsHandlerTests: XCTestCase {
         let stopCommand2 = try XCTUnwrap(commandSubscriber.receivedCommands[3] as? RUMStopViewCommand)
         let startCommand3 = try XCTUnwrap(commandSubscriber.receivedCommands[4] as? RUMStartViewCommand)
 
-        XCTAssertTrue(startCommand1.identity.equals(view1))
-        XCTAssertTrue(stopCommand1.identity.equals(view1))
-        XCTAssertTrue(startCommand2.identity.equals(view2))
-        XCTAssertTrue(stopCommand2.identity.equals(view2))
-        XCTAssertTrue(startCommand3.identity.equals(view1))
+        XCTAssertTrue(startCommand1.identity.equals(view1.asRUMViewIdentity()))
+        XCTAssertTrue(stopCommand1.identity.equals(view1.asRUMViewIdentity()))
+        XCTAssertTrue(startCommand2.identity.equals(view2.asRUMViewIdentity()))
+        XCTAssertTrue(stopCommand2.identity.equals(view2.asRUMViewIdentity()))
+        XCTAssertTrue(startCommand3.identity.equals(view1.asRUMViewIdentity()))
```

Comment:
nit/ We could avoid these changes (they made most of this PR) if we add convenience extension for tests.